hatchbuck/objects.py

```diff
@@ -109,6 +108,10 @@ def __init__(self, api_key, json_data=None):
     def add_email(self, address, type):
         self.emails.append(Email({'address': address, 'type': type}))
 
+    def add_customField(self, name, value):
```

Comment:
For consistency with other methods in this class, we should use snake case for the method name: `add_custom_field`

Reply:
Sure! That make sense.